checker/jtreg/stubs/issue1496/ClassAnnotation.astub

```diff
@@ -0,0 +1,10 @@
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.framework.qual.DefaultQualifier;
+
+// This checks that fully-qualified class name in annotation properly handled by the StubParser.
+@DefaultQualifier(org.checkerframework.checker.nullness.qual.Nullable.class)
```

Comment:
Can you use `org.checkerframework.checker.nullness.qual.NonNull.class` here, to make sure that the fully-qualified class works even without an import?